relay-server/src/utils/profile.rs

```diff
@@ -0,0 +1,77 @@
+use failure::Fail;
+
+use android_trace_log::AndroidTraceLog;
+use serde::{Deserialize, Serialize};
+
+use crate::envelope::{ContentType, Item};
+
+pub const ANDROID_SDK_NAME: &str = "sentry.java.android";
+
+#[derive(Serialize, Deserialize)]
+struct AndroidProfile {
+    android_api_level: u16,
+    build_id: String,
+    device_locale: String,
+    device_manufacturer: String,
+    device_model: String,
+    device_os_name: String,
+    device_os_version: String,
+    environment: String,
+    error_code: String,
+    error_description: String,
+    platform: String,
+    stacktrace: String,
+    stacktrace_id: String,
+    trace_id: String,
+    transaction_id: String,
+    transaction_name: String,
+    version_code: String,
+    version_name: String,
+
+    android_trace: Option<AndroidTraceLog>,
```

Comment:
Allow the field to be _missing_:

```suggestion
    #[serde(default)]
    android_trace: Option<AndroidTraceLog>,
```

Are any other fields optional? Mark all of them with `#[serde(default)]` which uses the type's `Default::default()` value in case the field is missing in the original payload.

Follow-up question: Would you like any field to be omitted from output if its empty?

Reply:
No, I want all the fields populated, even if they are empty.